docs/pages/release_notes.rst

```diff
@@ -3,6 +3,46 @@
 Release notes
 #############
 
+************************
+TBD (v6.0.0)
```

Comment:
Peanut

Reply:
A peanut is botanically speaking not a nut, but okay..

Reply:
We focus on what you can buy at the Nuts stand.